manifests/postgresql-operator-default-configuration.yaml

```
@@ -5,6 +5,7 @@ metadata:
 configuration:
   # enable_crd_validation: true
   etcd_host: ""
+  # kubernetes_use_configmaps: "False"
```

```suggestion
  # kubernetes_use_configmaps: false
```
here it's the opposite :smiley: